CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+## [1.19.1] - 2020-08-03
+### Fixed
+ * Move OS-specific logic into OS-specific files to avoid compile issues on
+   non-Unix platforms.
```

Comment:
Relay changes aren't worth mentioning?

Reply:
Yeah, typically we don't mention relay changes in the changelog since it's user-facing.

The only time we mention them is if nothing else changes in that version.